### PR TITLE
GH-1015 Fix issue

### PR DESCRIPTION
**Pull request by MouazAlzahabi**

### CODE REVIEW

File: Multisig/Logic/Models/AppSettings.swift [hunk 1]
```diff
@@ -39,6 +39,18 @@ extension AppSettings {
     @AppSetting(\.displayMode)
     static var displayMode: Int32
 
+    @AppSetting(\.fiatCode)
+    private static var fiatCode: String?
```
Comment (DmitryBespalov) [inline Multisig/Logic/Models/AppSettings.swift:43]: what is the difference between fiat code and selected fiat code? I see there's a default value, but maybe we could extend the property wrapper to accept a default value and not have 2 properties for the same thing?

File: Multisig/Logic/Models/TokenBalance.swift [hunk 2]
```diff
@@ -18,29 +18,29 @@ struct TokenBalance: Identifiable, Hashable {
...
 extension TokenBalance {
-    init(_ response: SafeBalancesRequest.Response) {
+    init(_ response: SafeBalancesRequest.Response, code: String) {
         self.init(address: response.tokenAddress?.address ?? .ether,
                   symbol: response.token?.symbol,
                   logoUri: response.token?.logoUri,
                   tokenBalance: response.balance,
                   decimals: response.token?.decimals,
-                  fiatBalance: response.balanceUsd)
+                  fiatBalance: response.balanceUsd, code: code)
```
Comment (DmitryBespalov) [inline Multisig/Logic/Models/TokenBalance.swift]: code should be on the next line

File: Multisig/Logic/Models/TokenBalance.swift [hunk 3]
```diff
@@ -18,29 +18,29 @@ struct TokenBalance: Identifiable, Hashable {
...
 
-    init(_ item: SCGBalance) {
+    init(_ item: SCGBalance, code: String) {
         self.init(address: item.tokenInfo.address.address,
                   symbol: item.tokenInfo.symbol,
                   logoUri: item.tokenInfo.logoUri,
                   tokenBalance: item.balance,
                   decimals: item.tokenInfo.decimals,
-                  fiatBalance: item.fiatBalance)
+                  fiatBalance: item.fiatBalance, code: code)
```
Comment (DmitryBespalov) [inline Multisig/Logic/Models/TokenBalance.swift]: code should be on the next line

**Review by DmitryBespalov — APPROVED** [review]

File: Multisig/Logic/Models/AppSettings.swift (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/Logic/Models/AppSettings.swift:43]: I couldn't updated it now, it seems this will need refactor for were we used it

**Pull request merged**